### PR TITLE
Dwarven Ethnicity Rework

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Reworked the old ethnicity used by Dwarves to be closer to how they are depicted in most Warcraft media.
- Greater focus on making female dwarves look prettier. 😳

# How to test:
1. Open up Ruler Designer and generate male and female **_Dwarves_** using the reworked ethnicities (`dwarven_rework`, `frostborn_rework`, `bronzebeard_rework`, `dark_iron_rework`, `wildhammer_rework`) while comparing them to the old ethnicities higher up on the list of ethnicities.
2. You may also load up a game and look for **_Dwarves_** in-game to see how they look when animated.

Provide feedback on the reworked ethnicity and whether it's an improvement over the old or not.

### CODE REVIEW

**Comment by ercarp** [comment]
**A few examples of randomly-generated dwarves using the reworked ethnicity:**

<details><summary>Click to expand</summary>

image
image
image
image
image
image
image
image

</details>

File: common/ethnicities/wc_ethnicities_dwarven_rework.txt [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+dwarven_rework = {
+	template = "caucasian"
+	
+	skin_color = {
+		1 = { 0.047 0.082 0.122 0.263 }
+	}
+	eye_color = {
+		1 = { 0 0.300 0.757 0.600 } #Most colors
+		1 = { 0.914 0.408 1 0.600 } #Pink
+	}
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_dwarven_rework.txt:55]: I would expect dwarves to be the same eye colors as humans.
I'm not aware of lore characters with bright-colored eyes, so I assume it's player customization from the WoW.

Comment (ercarp) [reply common/ethnicities/wc_ethnicities_dwarven_rework.txt:55]: Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.

Comment (zumbak04) [reply common/ethnicities/wc_ethnicities_dwarven_rework.txt:55]: > Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.

I would give them the same eye colors humans have.

**Review by TheDrPibb1999 — APPROVED** [review]
I personally like the reworks :), they look great to me!!! good work

File: common/ethnicities/wc_ethnicities_dwarven_rework.txt (hunk 1 above)
Comment (ercarp) [reply common/ethnicities/wc_ethnicities_dwarven_rework.txt]: > 
> 
> > Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.
> 
> I would give them the same eye colors humans have.

Changed this in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/200/commits/344a4ddb36c71ee4b4699e9361c6ac2cd533a9ee.

File: common/ethnicities/wc_ethnicities_dwarven_rework.txt [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+	template = "caucasian"
+	
+	skin_color = {
+		1 = { 0.047 0.082 0.122 0.263 }
+	}
+	eye_color = {
+		#1 = { 0 0.300 0.757 0.600 } #Most colors
+		#1 = { 0.914 0.408 1 0.600 } #Pink
+		1 = { 0.129 0.396 0.776 0.788 }
+	}
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_dwarven_rework.txt:56]: If you want the human eyes, you can just remove this entity. The dwarves will take eye colors from humans they're based on.
```suggestion
```

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**